package.json

```
@@ -47,7 +47,7 @@
 		"karma-sinon": "^1.0.5",
 		"karma-webpack": "^4.0.2",
 		"mocha": "^9.2.1",
-		"rollup": "^2.47.0",
+		"rollup": "^2.72.0",
```

```suggestion
		"rollup": "^2.75.3",
```

There is a newer version of `rollup`.